spec-main/api-crash-reporter-spec.ts

```diff
@@ -183,6 +183,24 @@ ifdescribe(!process.mas && !process.env.DISABLE_CRASH_REPORTER_TESTS && process.
     }
   })
 
+  describe('getUploadedReports', () => {
```

Comment:
Maybe move this test up higher in the execution order to ensure that a previous test didn't start the crash reporter?

Reply:
I dug into this and unfortunately it looks like the smoke tests do it somehow, so i don't think it'll be possible to replicate the environment needed to throw the error :(